libs/d3d12/main.c

```diff
@@ -233,39 +197,46 @@ static VkPhysicalDevice d3d12_get_vk_physical_device(struct vkd3d_instance *inst
     if ((vr = pfn_vkEnumeratePhysicalDevices(vk_instance, &count, vk_physical_devices)) < 0)
         goto done;
 
-    if (pfn_vkGetPhysicalDeviceProperties2)
+    TRACE("Matching adapters by LUIDs.\n");
+
+    for (i = 0; i < count; ++i)
     {
-        TRACE("Matching adapters by LUIDs.\n");
+        pfn_vkGetPhysicalDeviceProperties(vk_physical_devices[i], &properties2.properties);
 
-        for (i = 0; i < count; ++i)
-        {
-            memset(&id_properties, 0, sizeof(id_properties));
-            id_properties.sType = VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_ID_PROPERTIES;
+        /* Skip over < 1.1 physical devices because... 1.1 instances
+         * still expose those apparently... */
+        if (properties2.properties.apiVersion < VK_API_VERSION_1_1)
```

Comment:
Should probably check `VKD3D_MIN_API_VERSION` (which is currently 1_1 but may be bumped to 1_2 at some point) and remove the comment above, instead WARN that we skipped an adapter.

Reply:
Resolved.